includes/gateway/class-omise-payment-creditcard.php

```diff
@@ -379,13 +379,11 @@ public function process_capture( $order ) {
 			} catch ( Exception $e ) {
 				$this->order()->add_order_note(
 					sprintf(
-						wp_kses(
-							__( 'Omise: Payment failed (manual capture).<br/>%s', 'omise' ),
-							array( 'br' => array() )
-						),
+						wp_kses( __( 'Omise: Payment failed (manual capture).<br/>%s', 'omise' ), array( 'br' => array() ) ),
```

Comment:
Note, this line is just to remove unnecessary breakline to make it a little more easy to read. Nothing change in term of function.